packages/edit-site/src/components/global-styles-sidebar/style.scss

```diff
@@ -1,22 +1,3 @@
-.editor-sidebar {
-	width: $sidebar-width;
-
-	> .components-panel {
-		border-left: 0;
-		border-right: 0;
-		margin-bottom: -1px;
-		margin-top: -1px;
-
-		> .components-panel__header {
-			background: $gray-100;
-		}
-	}
-
-	.block-editor-block-inspector__card {
-		margin: 0;
-	}
-}
```

Comment:
I checked these styles one by one. None of them are needed:

1. `width: $sidebar-width;` sets the same width 280px.
2. `border-left: 0; border-right: 0;` are not needed because `.components-panel` has `border: none` now. 
3. `margin-bottom: -1px; margin-top: -1px;` are most likely a trick to collapse the double borders. But there is no border now because of the aforementioned `border: none`. 
4. `background: $gray-100;` This one does change the background from the original `#fff`, to `#f0f0f0`, which I found a source of inconsistency.
5. `block-editor-block-inspector__card`. There is no such element anymore.